/home/users/mashka/schrodinger.ve/lib/python3.6/site-packages/pandas/compat/__init__.py:117: UserWarning: Could not import the lzma module. Your installed Python is incomplete. Attempting to use lzma compression will result in a RuntimeError.
  warnings.warn(msg)
Traceback (most recent call last):
  File "/home/users/mashka/combind/combind", line 500, in <module>
    main()
  File "/share/software/user/restricted/schrodinger/2019-2/internal/lib/python3.6/site-packages/click/core.py", line 764, in __call__
    return self.main(*args, **kwargs)
  File "/share/software/user/restricted/schrodinger/2019-2/internal/lib/python3.6/site-packages/click/core.py", line 717, in main
    rv = self.invoke(ctx)
  File "/share/software/user/restricted/schrodinger/2019-2/internal/lib/python3.6/site-packages/click/core.py", line 1137, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/share/software/user/restricted/schrodinger/2019-2/internal/lib/python3.6/site-packages/click/core.py", line 956, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/share/software/user/restricted/schrodinger/2019-2/internal/lib/python3.6/site-packages/click/core.py", line 555, in invoke
    return callback(*args, **kwargs)
  File "/home/users/mashka/combind/combind", line 317, in featurize
    features.compute_single_features(poseviewers, processes=processes)
  File "/home/users/mashka/combind/features/features.py", line 116, in compute_single_features
    if type(pvs[0]) == list:
IndexError: list index out of range
